tests/test_actions.py

```diff
@@ -234,6 +234,12 @@ def repomod_testing(self, repo_type):
         self.assertNotIn("file:///sample_rpms", self.action_stdout)
         self.assertIn("file:///some/new/url", self.action_stdout)
 
+        self.assertEqual(self.call_action_ordered_args("repomod", [
+            "sample_repo", "another_repo", "--gpgcheck enable"]), 0)
```

Comment:
You should also check whether the changes are actually applied with `assertNotIn`/`assertIn`, or with whatever other assert there is.

Reply:
Added.